sympy/concrete/delta.py

```diff
@@ -1,5 +1,13 @@
-from __future__ import print_function, division
+"""
+This module implements Kronecker Delta function.
```

Comment:
The delta function itself is in sympy.functions. This module has functions to compute sums and products containing the Kronecker Delta function.